.github/workflows/ci.yml

```
@@ -26,11 +26,15 @@ jobs:
           key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
           restore-keys: |
             ${{ runner.os }}-nuget-
-
-      - name: Setup dotnet
+      
+      - name: Setup dotnet 3.1 
```

Becareful of visual studio and it's tendency to add whitespace.
```suggestion

      - name: Setup dotnet 3.1
```

---

Fixed in 4d342b4d371792e2da0622f357b6e77f9295200c

---

Nope 😆